core/src/test/java/org/mule/processor/DelegateTransactionTestCase.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.processor;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Answers.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.when;
+
+import org.mule.api.MuleContext;
+import org.mule.tck.junit4.AbstractMuleTestCase;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+
+
+public class DelegateTransactionTestCase extends AbstractMuleTestCase
+{
+
+    public static final int DEFAULT_TX_TIMEOUT = 2;
+    private MuleContext mockMuleContext = Mockito.mock(MuleContext.class, RETURNS_DEEP_STUBS.get());
```

Comment:
this should be initialised on a before() method. Each test should have a brand new mock instance. Additionally, use static imports